app/src/main/java/com/ably/tracking/demo/subscriber/ui/bottomsheet/LocationUpdateBottomSheet.kt

```diff
@@ -70,29 +71,13 @@ fun LocationUpdateBottomSheetContentPlaceholder() = AATSubscriberDemoTheme {
 @Composable
 @Preview
 fun LocationUpdateBottomSheetContent(
-    @PreviewParameter(LocationUpdatePreview::class) locationUpdateBottomSheetData: LocationUpdateBottomSheetData
+    @PreviewParameter(LocationUpdatePreview::class)
+    locationUpdateBottomSheetData: LocationUpdateBottomSheetData
 ) = AATSubscriberDemoTheme {
-    val location = locationUpdateBottomSheetData.locationUpdate?.location
     Column(
         modifier = Modifier.fillMaxWidth()
     ) {
-        mapOf(
-            R.string.trackable_location_time to location?.time?.formatToDate(),
-            R.string.trackable_location_latitude to location?.latitude,
-            R.string.trackable_location_longitude to location?.longitude,
-            R.string.trackable_location_speed to location?.speed,
-            R.string.trackable_location_bearing to location?.bearing,
-            R.string.trackable_location_altitude to location?.altitude,
-            R.string.trackable_location_accuracy to location?.accuracy,
-            R.string.trackable_location_last_interval to
-                    locationUpdateBottomSheetData.lastLocationUpdateInterval?.toIntervalWithUnit(),
-            R.string.trackable_location_average_interval to
-                    locationUpdateBottomSheetData.averageLocationUpdateInterval?.toIntervalWithUnit(),
-            R.string.trackable_location_desired_interval to
-                    locationUpdateBottomSheetData.resolution?.desiredInterval?.toIntervalWithUnit(),
-            R.string.trackable_location_skipped_locations to
-                    locationUpdateBottomSheetData.locationUpdate?.skippedLocations?.size,
-        )
+        cha(locationUpdateBottomSheetData)
```

Comment:
What does `cha` stand for? For me it's not obvious what this method does :thinking:

Reply:
Thanks, that was development leftover

Reply:
https://github.com/ably/asset-tracking-android-customer-app-demo/pull/15/commits/b93239d6a2cb6c0ca339c3f8df56f2342f634203